components/feature/awesomebar/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="switch_to_tab_description">Switch to tab</string>
```

Comment:
nit: Can you please add a comment in front of this element, explaining the context this string shows up in? The translation tools will pick that up and show that to our translators.